tests/integration/api/test_aws_integration.py

```diff
@@ -42,6 +46,9 @@ def test_list(self):
             account_id=TEST_ACCOUNT_ID_2,
             role_name=TEST_ROLE_NAME
         )
+        self.accounts_to_cleanup.append({"account_id": TEST_ACCOUNT_ID_1, "role_name": TEST_ROLE_NAME})
```

Comment:
Since we have all account ids being used defined as constants, can we just use a teardown method that would clean all accounts all the time? This way we don't have to remember to append to `accounts_to_cleanup`.

Reply:
Yeah we can do that. I am planning on using this `defer`-like approach to the `roles` endpoint where you don't know the ID until after you call create. But since we know the `id`s here, we can just delete all between test runs.